app/commands/user/reputation_token/award_for_pull_request_author.rb

```diff
@@ -0,0 +1,49 @@
+class User
+  class ReputationToken
+    class AwardForPullRequestAuthor
+      include Mandate
+
+      initialize_with :params
+
+      def call
+        return unless has_author?
+        return unless merged?
+
+        user = User.find_by(github_username: params[:author_username])
```

Comment:
I could encapsulate this in a memoized method if you feel like that would make more sense